### ISSUE TITLE
Misformatted description in translated ".po" files for base modules (All Languages ?)

### ISSUE THREAD

**Issue by Yotouille** [odoo/odoo#13695]
**Impacted versions**: (there may be more)
  - 9.0c-20160726 (Community Edition)
  - 9.0e-20160901 (Enterprise Edition)

And maybe all `*.po` files in: `/usr/lib/python2.7/dist-packages/openerp/addons/base/i18n/` directory.

**Steps to reproduce**:
This is the best example for it crash instantly (step 2 and 5 optional here), but you can test with random language/module to see noises in server logs.
  1. As an admin, load and use french language
  2. `tail -f` on your server logs
  3. Search for `auth_ldap` or `webkit` in modules
  4. Click on module to generate description
  5. Go back on the logs to see full Traceback

**Current behavior**:
The description is not shown and a Traceback appear in webUI, for the translated string is misformatted in `fr.po`. These are the two worst cases I could find, but playing with others languages and generating base modules description randomly, I could see there is almost always a warning in the logs (most are `(WARNING/2) Title underline too short`) and a Traceback following.

**Expected behavior**:
  - The best thing would be to always have a good formatted description and no noise in server logs
  - Maybe some kind of a test to validate the imported description and load the default one (in english) if the test fail, ex: `(SEVERE/4)`.

**Log example**:
  - German / `auth_ldap` (Description IS shown on WebUI)
```
2016-10-06 12:56:58,747 1 INFO walou werkzeug: 172.17.0.1 - - [06/Oct/2016 12:56:58] "POST /web/dataset/call_kw/ir.module.module/fields_view_get HTTP/1.1" 200 -
<string>:2: (WARNING/2) Title underline too short.

Fügt Unterstützung zur Authentifizierung durch LDAP-Server hinzu.
========================================================
Traceback (most recent call last):
  File "/usr/lib/python2.7/logging/__init__.py", line 859, in emit
    msg = self.format(record)

  File "/usr/lib/python2.7/logging/__init__.py", line 732, in format
    return fmt.format(record)

  File "/usr/lib/python2.7/dist-packages/openerp/netsvc.py", line 102, in format
    return logging.Formatter.format(self, record)

  File "/usr/lib/python2.7/logging/__init__.py", line 474, in format
    s = self._fmt % record.__dict__

UnicodeDecodeError: 'ascii' codec can't decode byte 0xfc in position 188: ordinal not in range(128)
Logged from file module.py, line 115
2016-10-06 12:56:58,952 1 INFO walou werkzeug: 172.17.0.1 - - [06/Oct/2016 12:56:58] "POST /web/dataset/call_kw/ir.module.module/read HTTP/1.1" 200 -
<string>:2: (WARNING/2) Title underline too short.
```

  - French / `report_webkit`  (Description NOT shown on WebUI > Crash > Traceback)
```
2016-10-06 13:57:33,553 1 INFO walou werkzeug: 172.17.0.1 - - [06/Oct/2016 13:57:33] "POST /web/dataset/call_kw/ir.module.module/fields_view_get HTTP/1.1" 200 -
<string>:3: (WARNING/2) Title underline too short.

Module ajoutant un nouveau moteur de rapport fondé sur la bibliothèque WebKit (wkhtmltopdf) pour prendre en compte les rapports conçus en HTML + CSS.
=====================================================================================================================
<string>:8: (SEVERE/4) Unexpected section title.

Le module permet :
------------------
2016-10-06 13:57:33,773 1 ERROR walou openerp.http: Exception during JSON request handling.
Traceback (most recent call last):

.......["Traceback TRUNCATED tooooo long ! Only keeping the end." yotouille]........

  File "/usr/lib/python2.7/dist-packages/docutils/statemachine.py", line 239, in run
    context, state, transitions)

  File "/usr/lib/python2.7/dist-packages/docutils/statemachine.py", line 460, in check_line
    return method(match, context, next_state)

  File "/usr/lib/python2.7/dist-packages/docutils/parsers/rst/states.py", line 2721, in underline
    source=src, line=srcline)

  File "/usr/lib/python2.7/dist-packages/docutils/utils/__init__.py", line 235, in severe
    return self.system_message(self.SEVERE_LEVEL, *args, **kwargs)

  File "/usr/lib/python2.7/dist-packages/docutils/utils/__init__.py", line 193, in system_message
    raise SystemMessage(msg, level)

SystemMessage: <string>:8: (SEVERE/4) Unexpected section title.

Le module permet :
------------------
2016-10-06 13:57:33,792 1 INFO walou werkzeug: 172.17.0.1 - - [06/Oct/2016 13:57:33] "POST /web/dataset/call_kw/ir.module.module/read HTTP/1.1" 200 -
```

Coming from there: https://github.com/odoo/odoo/issues/12883

**Comment by Yotouille** [comment]
I wanted to add, as I was saying on the other issue, if a french newcomer (non-python-friendly) is using odoo in his language for the first time, he will certainly thinks these modules are broken. There is nothing clear about a description format issue related to some language (until you dig down a bit), and he may not try to use another language (what for ?).

**Comment by mart-e** [comment]
Thanks for the detailed report.
I have fixed the translation at 3f192abafbe4ffbccaba07a5f050419479d7fb7a
